frontend/src/scenes/funnels/FunnelHistogram.tsx

```diff
@@ -23,26 +25,33 @@ export function FunnelHistogramHeader(): JSX.Element | null {
             <span className="funnel__header__steps__label">Steps</span>
             {histogramStepsDropdown.length > 0 && (
                 <Select
-                    defaultValue={histogramStepsDropdown[0]?.value}
-                    onChange={changeHistogramStep}
+                    defaultValue={histogramStepsDropdown[0]?.from_step}
+                    onChange={(from_step) => {
+                        changeHistogramStep(from_step, from_step + 1)
+                    }}
                     dropdownMatchSelectWidth={false}
+                    dropdownAlign={ANTD_TOOLTIP_PLACEMENTS.bottomLeft}
                     data-attr="funnel-bar-layout-selector"
                     optionLabelProp="label"
                 >
                     {histogramStepsDropdown.map((option, i) => {
                         const basisStep = getReferenceStep(stepsWithCount, stepReference, i)
                         return (
-                            <Select.Option key={option?.value} value={option?.value || 1} label={<>{option?.label}</>}>
+                            <Select.Option
+                                key={option?.from_step}
+                                value={option?.from_step}
+                                label={<>{option?.label}</>}
+                            >
                                 <Col style={{ minWidth: 300 }}>
                                     <Row style={{ justifyContent: 'space-between', padding: '8px 0px' }}>
                                         <span className="l4">{option?.label}</span>
                                         <span className="text-muted-alt">
-                                            Mean time: {humanFriendlyDuration(option?.average_conversion_time)}
+                                            Mean time: {humanFriendlyDuration(option.average_conversion_time)}
                                         </span>
                                     </Row>
                                     <Row className="text-muted-alt">
                                         Total conversion rate:{' '}
-                                        {humanizeNumber(calcPercentage(option.count, basisStep.count))}%
+                                        {humanizeNumber(calcPercentage(option.count || 0, basisStep.count))}
```

Comment:
I believe "%" is missing? Super minor nit: `option.count ?? 0`

Reply:
Ah interesting, didn't know that there was a difference between || and ??, but it looks like ?? catches only `null` and `undefined` while || catches those and '' and 0. I guess there's no difference in the compiled code, but ?? would be more correct